tests/unit/common/test_utils.py

```diff
@@ -155,18 +155,18 @@ def test_open_path_with_atomic_rename(tmpdir):
     f1_path = f"{tmpdir}/f1"
     with utils.open_path_with_atomic_rename(f1_path) as f1:
         f1.write(b"test1")
-    assert Path(f1_path).read_text() == "test1"
+    assert Path(f1_path).read_text(encoding="utf-8") == "test1"
```

Comment:
I think it's because pylintrc `unspecified-encoding` warning, but this has only value for cross platform code.
I don't think we care about that for astacus, we could just disable the warning entirely.

Reply:
Done